From c5a117faafbe1cc6de4200315ba400bdc0c83e40 Mon Sep 17 00:00:00 2001
From: Robert Church <robert.church@windriver.com>
Date: Sat, 15 May 2021 16:16:41 -0400
Subject: [PATCH 7/8] Update postgres liveness check to support IPv6 addresses

Templating will add square brackets for IPv6 addresses which are
interpreted as an array vs. a string. Quote this so that it interpreted
correctly.

Signed-off-by: Robert Church <robert.church@windriver.com>
---
 charts/armada/templates/deployment-api.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/charts/armada/templates/deployment-api.yaml b/charts/armada/templates/deployment-api.yaml
index 2b65494..5c4825c 100644
--- a/charts/armada/templates/deployment-api.yaml
+++ b/charts/armada/templates/deployment-api.yaml
@@ -34,7 +34,7 @@ exec:
   - -vz
   - -w
   - "1"
-  - {{ .Values.conf.tiller.sql_endpoint_ip}}
+  - "{{ .Values.conf.tiller.sql_endpoint_ip }}"
   - "5432"
 {{- end }}
 
-- 
2.34.1

